MotorDriver.cpp

```diff
@@ -0,0 +1,41 @@
+#include MotorDriver.h
+#include <avr/io.h>
+#include <util/delay.h>
+
+MotorDriver::MotorDriver()
```

Comment:
Ask user to which pin to use with motors. Users has to specify their pin.
For this, create a convertion between avr pins and arduino pins.